timemachine/cpp/src/kernels/k_nonbonded_pairs.cuh

```diff
@@ -0,0 +1,200 @@
+#include "../fixed_point.hpp"
+#include "k_fixed_point.cuh"
+
+template <typename RealType>
+void __global__ k_nonbonded_pairs(
+    const int M, // number of pairs
```

Comment:
Not sure if we have a convention for this already?

Reply:
used to be called E, but I think anything works

Reply:
Makes sense; I had assumed we used `E` for "exclusions", and in that case we'd probably want something else for the general case